app/app.go

```diff
@@ -118,14 +118,20 @@ import (
 	appparams "github.com/umee-network/umee/v3/app/params"
 	"github.com/umee-network/umee/v3/swagger"
 	"github.com/umee-network/umee/v3/util/genmap"
-	uibctransfer "github.com/umee-network/umee/v3/x/ibctransfer"
-	uibctransferkeeper "github.com/umee-network/umee/v3/x/ibctransfer/keeper"
+
 	"github.com/umee-network/umee/v3/x/leverage"
 	leveragekeeper "github.com/umee-network/umee/v3/x/leverage/keeper"
 	leveragetypes "github.com/umee-network/umee/v3/x/leverage/types"
 	"github.com/umee-network/umee/v3/x/oracle"
 	oraclekeeper "github.com/umee-network/umee/v3/x/oracle/keeper"
 	oracletypes "github.com/umee-network/umee/v3/x/oracle/types"
+
+	uibctransfer "github.com/umee-network/umee/v3/x/ibctransfer"
```

Comment:
maybe we should call the module `uibc`? This way we would avoid renames and confusions? WDYT?
```
"github.com/umee-network/umee/v3/x/ubic"
ics20keeper "github.com/umee-network/umee/v3/x/uibc/ics20/keeper"
...
```

Reply:
Okay , I will rename `ibctransfer` to `uibc`